sixMans/game.py

```diff
@@ -377,28 +379,45 @@ async def process_self_picking_teams(self, emoji, user, added=True):
             await self.update_game_info()
             await self._notify(Strings.ONGOING_GS)
 
+    async def _remove_stale_reactions(self, emoji_hex: int, member: discord.Member):
+        if self.reaction_lock:
+            return
+
+        self.reaction_lock = True
+        coros = []
+
+        for react_hex, reaction in [(react_hex, self._get_pick_reaction(react_hex)) for react_hex in SELECTION_MODES]:
+            if react_hex != emoji_hex:
+                coros.append(self.info_message.remove_reaction(reaction, member))
+
+        await asyncio.gather(*coros)
+        self.reaction_lock = False
```

Comment:
Could you explain what this function does, and what purpose it serves?

Reply:
This function removes any reactions from the member that are not the emoji_hex.
It's locked to avoid excessive calls from people spamming reactions.